modules/rds__explore_snapshots/main.py

```diff
@@ -63,10 +63,12 @@ def cleanup(pacu):
     success = True
     for instance in data['Instances']:
         client = pacu.get_boto3_client('rds', data['Instances'][instance]['AvailabilityZone'][:-1])
-        delete_instance(client, instance, pacu.print)
+        if not delete_instance(client, instance, pacu.print):
```

Comment:
Will this encounter errors/handle them? Same question for line 70

Reply:
The delete_instances function will handle errors related to deleting an instance and print out messages accordingly. success should be set to false upon any issues, but attempts to delete the following temporary resources will continue until all attempts have been made. for each resource.